types/lru-cache/index.d.ts

```diff
@@ -1,23 +1,127 @@
-// Type definitions for lru-cache 4.1
+// Type definitions for lru-cache 5.0
```

Comment:
Would you mind upgrading this directly to `5.1` and adding the `updateAgeOnGet` option to the `Options` interface?

Reply:
Ok. I also think to return `v4` to separate folder (like in Node.js types), because some other definitions depend from it.